app/src/main/kotlin/com/wire/android/ui/home/settings/account/MyAccountViewModel.kt

```diff
@@ -67,41 +70,44 @@ class MyAccountViewModel @Inject constructor(
     var myAccountState by mutableStateOf(MyAccountState())
         private set
 
+    @VisibleForTesting
+    var hasSAMLCred by Delegates.notNull<Boolean>()
+
+    @VisibleForTesting
+    var managedByWire by Delegates.notNull<Boolean>()
```

Comment:
Life is not fair 😓  good solution to the problem of verifying these 2 conditions.